src/presence_analyzer/views.py

```diff
@@ -47,6 +48,22 @@ def serve_template(template):
         return make_response('page not fond', 404)
 
 
+@app.route('/api/v1/photo_url/<int:user_id>', methods=['GET'])
+@jsonify
+def prepare_photo_url(user_id):
+    """
+    Returns url for intranet api in order to get photo of given user.
+    """
+    conf = get_server_config()
+    return '{}{}{}{}{}'.format(
+        conf['protocol'],
+        '://',
```

Comment:
Move it up to string, only things that u get like conf['protocol'] should be in .format().